deploy/deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
how about adding ```set -e``` ? (http://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#The-Set-Builtin)

Reply:
+1
ignoring errors is impolite.